### PR TITLE
feat: add custom lacework audit policy

### PR DESCRIPTION
**Pull request by hazedav**
Introducing lacework audit policy

ALLY-654

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -40,6 +41,12 @@ variable "lacework_integration_name" {
   description = "The name of the integration in Lacework"
 }
 
+variable "lacework_audit_policy_name" {
+  type    = string
+  default = ""
+  description = "The name of the custom audit policy (which extends SecurityAudit) to allow Lacework to read configs.  Defaults to lwaudit-policy-$${random_id.uniq.hex} when empty"
```
Comment (afiune) [inline variables.tf:47]: Not sure you can do that. 🤨

**Review by afiune — APPROVED** [review]
Just one doubt, please double check that the description of the new variable works as expected, if so; I'm +1

File: variables.tf (hunk 1 above)
Comment (hazedav) [reply variables.tf:47]: This is done via `locals` in `main.tf`.  This description is `$$` escaped string literal.

**Review by alannix-lw — APPROVED** [review]
LGTM! Tested upgrading in my lab as well, and no issues. 👍

**Pull request merged**